open_city_profile/models.py

```diff
@@ -0,0 +1,119 @@
+from django.db import models
+
+
+class SerializableMixin(models.Model):
```

Comment:
maybe this can be it `utils/models.py`? since it already exists

Reply:
Very much true, missed this somehow 😲